### ISSUE TITLE
VK_EXT_host_image_copy incompatible with VK_KHR_external_memory_win32

### ISSUE THREAD

**Issue by Agrael1** [KhronosGroup/Vulkan-Docs#2431]
Is VK_EXT_host_image_copy incompatible with VK_KHR_external_memory_win32?
When I create VkImage with `VkImageCreateInfo::usage = VK_IMAGE_USAGE_HOST_TRANSFER_BIT_EXT` it succeeds, however if I call `vkGetImageMemoryRequirements2KHR` on such image it says `memoryTypeBits` is equal to 0. Any other type is allocated successfully, but if I have external memory as pNext to the VkImageCreateInfo
```
    constexpr static VkExternalMemoryImageCreateInfoKHR external_info{
        .sType = VK_STRUCTURE_TYPE_EXTERNAL_MEMORY_IMAGE_CREATE_INFO_KHR,
        .pNext = nullptr,
        .handleTypes = VK_EXTERNAL_MEMORY_HANDLE_TYPE_OPAQUE_WIN32_BIT
    };
```
it fails to find memory bits.

If I was to remove the usage `VK_IMAGE_USAGE_HOST_TRANSFER_BIT_EXT` the memory is found. If I remove `external_info` from pNext, the memory allocation succeeds, although with validation error.

Is this a driver bug, or `VK_KHR_external_memory_win32` has limits on what memory it can export?

I'm using NVidia A4000 with 560. driver on Windows 11

**Comment by pdaniell-nv** [comment]
Right, we don't currently support this, but I think we could. I'll investigate internally.

**Comment by pdaniell-nv** [comment]
Hi @Agrael1. We just pushed a new Vulkan beta driver version 596.46 for Windows and 595.44.06 for Linux: https://developer.nvidia.com/vulkan-driver. This contains a tweak that should allow external buffer and image objects to be bound to host-visible device-local memory. If you get a chance to give it a try, please let me know how it works out for you. Thanks!
